includes/Integrations/Elementor/Subscription_Plans_Widget.php

```
@@ -0,0 +1,1506 @@
+	/**
+	 * Render a single plan card
+	 *
+	 * @since WPUF_SINCE
+	 *
+	 * @param \WP_Post $pack              The subscription plan post object.
+	 * @param string   $current_pack_id   Current active pack ID.
+	 * @param string   $current_pack_status Current pack status.
+	 * @return void
+	 */
+	protected function render_plan_card( $pack, $current_pack_id, $current_pack_status ) {
+		$settings               = $this->get_settings_for_display();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$settings'. (undefined)

(UnusedLocalVariable)